iocore/net/QUICNetVConnection.cc

```diff
@@ -811,7 +811,7 @@ QUICNetVConnection::state_handshake(int event, Event *data)
   case QUIC_EVENT_PATH_VALIDATION_TIMEOUT:
     this->_handle_path_validation_timeout(data);
     break;
-  case EVENT_IMMEDIATE:
+  case VC_EVENT_INACTIVITY_TIMEOUT:
```

Comment:
After #5839, `InactivityCop` will send `VC_EVENT_ACTIVE_TIMEOUT` event too.
We don't need to handle it correctly on this PR, but it'd better adding a TODO comment.

Reply:
Which PR? This is #5839.

Reply:
oh, #5824. Which broke the quic.

Reply:
Ah, *ACTIVE*_TIMEOUT. Yeah, I realized that we don't have code for active timeout. I'll create an issue for it.